### PR TITLE
[IMP]cfg: bump pylint-odoo to v8.0.21

### PR DESCRIPTION
**Pull request by antonag32**
This adds the new pylint-odoo `deprecated-odoo-model-method` check.

### CODE REVIEW

File: src/pre_commit_vauxoo/cfg/.pre-commit-config-optional.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ default_language_version:
   node: "14.13.0"
 repos:
   - repo: https://github.com/OCA/pylint-odoo
-    rev: v8.0.20
+    rev: v8.0.21
```
Comment (moylop260) [inline src/pre_commit_vauxoo/cfg/.pre-commit-config-optional.yaml:32]: change both cfg files with the same version, please

**Comment by moylop260** [comment]
LGTM

(Let me know when "draft" is removed)

**Comment by antonag32** [comment]
@moylop260 I removed the draft, I was just waiting to test it on a project, I tested it on sbd and it looked good.

**Comment by moylop260** [comment]
@antonag32 

Even if was released the docker image was not built correctly:
 - https://git.vauxoo.com/devops/dockerv/-/jobs/1392730

@ruiztulio 

Could you help us here, please?

**Comment by ruiztulio** [comment]
@moylop260 This seems to be a problem with [cargo](https://github.com/rust-lang/cargo/issues/10303) and lingit2, the issue was reported [here](https://github.com/libgit2/libgit2/issues/5525).

Seems to be solved as for the author comment, so I have no idea what's going on, I will need more tome to check which dependency is not updated.

The error was originated by a change in a [Twine dependency](https://github.com/pypa/twine/issues/1015), they changed to [nh3]() which depends on [amonia](https://github.com/rust-ammonia/ammonia) and is the one raising the error.

I managed to replicate it:

image


Now I need to solve it (WIP).

**Comment by moylop260** [comment]
@ruiztulio 

What about freezing the `twine` version meanwhile it is fixed?

**Pull request merged**